packages/mongo/collection.js

```diff
@@ -24,93 +24,134 @@ The default id generation technique is `'STRING'`.
  * @param {Function} options.transform An optional transformation function. Documents will be passed through this function before being returned from `fetch` or `findOne`, and before being passed to callbacks of `observe`, `map`, `forEach`, `allow`, and `deny`. Transforms are *not* applied for the callbacks of `observeChanges` or to cursors returned from publish functions.
  * @param {Boolean} options.defineMutationMethods Set to `false` to skip setting up the mutation methods that enable insert/update/remove from client code. Default `true`.
  */
-Mongo.Collection = function (name, options) {
-  var self = this;
-  if (! (self instanceof Mongo.Collection))
-    throw new Error('use "new" to construct a Mongo.Collection');
-
-  if (!name && (name !== null)) {
-    Meteor._debug("Warning: creating anonymous collection. It will not be " +
-                  "saved or synchronized over the network. (Pass null for " +
-                  "the collection name to turn off this warning.)");
-    name = null;
-  }
+Mongo.Collection = class Collection {
+  constructor(name, options) {
```

Comment:
Since a lot of the changes in this file are just reindentation, you may want to [review the diff using `?w=1`](https://github.com/meteor/meteor/pull/9359/files?w=1).

Reply:
I really enjoy that [`refined-github`](https://github.com/sindresorhus/refined-github/pull/661), a Chrome Extension I picked up thanks to @stubailo, provides the "No Whitespace" button on PR review. Previously, with or without that extension, the `?w=1` was lost when going next/previous during commit-by-commit review but I submitted https://github.com/sindresorhus/refined-github/pull/661 to maintain the `?w=1` on navigation (`n`/`p`) and I've enjoyed it ever since!